sklearn/cluster/hierarchical.py

```diff
@@ -642,6 +642,7 @@ class AgglomerativeClustering(BaseEstimator, ClusterMixin):
           all observations of the two sets.
 
     pooling_func : callable, default=np.mean
+        deprecated, since AgglomerativeClustering do not use pooling_func
```

Comment:
I'd just say "ignored. Deprecate to be removed in ..." And remove the rest as irrelevant